src/main/java/core/basesyntax/dao/ReportWriterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.dao;
+
+import static core.basesyntax.model.Storage.storageOfFruits;
+
+import core.basesyntax.model.FruitTransactionParser;
+import core.basesyntax.model.Storage;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static FruitTransactionParser fruitTransactionParser = new FruitTransactionParser();
+    private Storage storage = new Storage();
+
+    @Override
+    public String generateReport(String fileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            StringBuilder builder = new StringBuilder();
+            builder.append("fruit,quantity" + System.lineSeparator())
+                    .append("banana, " + storageOfFruits.get("banana")
+                            + System.lineSeparator())
+                    .append("apple, " + storageOfFruits.get("apple"));
```

Comment:
You don't need to hardcode these values, instead you need to iterate through all keys and values in your storage, because it may consist other fruits as well

Reply:
How can I do this in another way?